src/sx-design/src/Button/Button.js

```diff
@@ -43,13 +49,21 @@ export default function Button(props: Props): React.Element<'button'> {
     });
   }
 
+  const handleOnClick = useCallback(
```

Comment:
Why do we need this `useCallback`?

Reply:
Like with the `Menu` PR (and https://github.com/adeira/universe/pull/3192) - the `useCallback` is incorrect here, thanks for pointing it out.